tests/core/scene/a-scene.test.js

```diff
@@ -162,6 +162,10 @@ suite('a-scene (without renderer)', function () {
 
       // mock canvas
       sceneEl.canvas = {
+        classList: {
+          add: function () {},
+          remove: function () {}
+        },
```

Comment:
@diarmidmackenzie now that #5176 is merged, you should remove first those 4 lines and then merge changes from master to fix the conflict.

Reply:
Of course -  sorry, didn't immediately register the implications of your comment.

Reply:
Those updates are done.